modules/virt-updating-multiple-vms.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-updating-multiple-vms_{context}"]
+= Updating multiple virtual machines
+
+You can update multiple virtual machines (VMs) at the same time by using the command line.
```

Comment:
```suggestion
You can use the command-line interface (CLI) to update multiple virtual machines (VMs) at the same time.
```

![Screenshot From 2025-05-08 14-43-39](https://github.com/user-attachments/assets/d2d5101f-d715-4984-af54-e095088beab8)

CLI is mentioned in the prereq so might be good to go with command-line interface.


![Screenshot From 2025-05-08 14-43-39](https://github.com/user-attachments/assets/570f93b3-7bde-4032-a5ab-aecdcb228cbf)

Guidance is not 100 % clearly, IMO, but the CLI is referenced in the prereq section.